src/App.js

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
-import Calculator from './components/Calculator';
-import Qoute from './components/qoute';
+import { Route, Routes } from 'react-router-dom';
+import CalculatorPage from './pages/Calculator';
+import Home from './pages/Home';
+import Navbar from './nav';
+import QoutePage from './pages/Qoute';
 
 function App() {
   return (
 
     <div className="App">
-      <div className="qoute_wrapper">
-        <Qoute />
-      </div>
+      <>
+        <Navbar />
+        <div className="container">
+          <Routes>
+            <Route path="/Home" element={<Home />} />
+            <Route path="/calculator" element={<CalculatorPage />} />
+            <Route path="/qoute" element={<QoutePage />} />
+          </Routes>
```

Comment:
- [OPTIONAL] To improve the UI, I would advise that you make the `Home` component display when the application starts. Currently, nothing is displayed when the application starts until I click a link in the Navigation pane. To implement this, change the route path to the `Home` component to `/` instead of `/Home`. 👍